packages/expo-modules-core/ios/ReactDelegates/RCTAppDelegate+Recreate.h

```diff
@@ -8,16 +8,21 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface RCTAppDelegate (Recreate)
 
-#if !TARGET_OS_OSX
 /**
  Recreates a root view bound with customized bundleURL, moduleName, initialProps, and launchOptions.
  If any of these parameters is null, the method will use the original one from `RCTAppDelegate` or `RCTRootViewFactory`.
  This method should be used with `EXReactRootViewFactory` that to recreate a root view.
  */
+#if !TARGET_OS_OSX
 - (UIView *)recreateRootViewWithBundleURL:(nullable NSURL *)bundleURL
                                moduleName:(nullable NSString *)moduleName
                              initialProps:(nullable NSDictionary *)initialProps
                             launchOptions:(nullable NSDictionary *)launchOptions;
+#else
+- (NSView *)recreateRootViewWithBundleURL:(nullable NSURL *)bundleURL
```

Comment:
the only difference here is UIView vs NSView?
could we import Platform.h (since we have the alias for UIView -> NSView) and remove this conditional declaration?
https://github.com/expo/expo/blob/4d5de4976f3dff8309b43ab750f2b3d4cc455644/packages/expo-modules-core/ios/Platform/Platform.h#L12

Reply:
Possibly, but this file no longer exists on main

Reply:
haha i tried to catch up prs that missed review from graphite and didn't notice the pr was months ago. thanks for sharing the info